cmd/agent/common/misconfig/global.go

```diff
@@ -0,0 +1,22 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package misconfig
+
+import "github.com/DataDog/datadog-agent/pkg/util/log"
+
+func ToLog() {
+	for name, check := range checks {
+		if err := check(); err != nil {
+			log.Warnf("misconfig: %s: %v", name, err)
```

Comment:
```suggestion
			log.Warnf("misconfig: %s: %w", name, err)
```

Reply:
I suspect it is a bit less subtle here - we aren't wrapping an error, just logging it as opposed to `errors.Wrapf`